examples/lock-app/lock-common/lock-app.matter

```diff
@@ -1662,6 +1662,30 @@ server cluster UserLabel = 65 {
   readonly attribute int16u clusterRevision = 65533;
 }
 
+/** Allows servers to ensure that listed clients are notified when a server is available for communication. */
+server cluster IcdManagement = 70 {
```

Comment:
So this is enabling this across all lock apps, right, including the Linux one we use for cert?

How dies it behave in the Linux lock-app, given lack of any obvious implementation in this PR?

Reply:
it uses the default values available in ChipConfig for the cluster. A part from that, adding the attributes of the cluster has no effect.